modules/ROOT/pages/logging.adoc

```diff
@@ -202,7 +202,7 @@ On MS Windows `logFilePath` supports the following path formats.
 [source,javascript]
 ----
 "C:/var/tmp/sglogfile.log"
-`C:\var\tmp\sglogfile.log`
+`C:\\var\\tmp\\sglogfile.log`
 `/var/tmp/sglogfile.log`
 "/var/tmp/sglogfile.log"
```

Comment:
@jamiltz I'm not sure if this is a valid Windows log file path or not. I can see how omitting the leading `/` would make it relative to SG, but can't say without testing.

Reply:
It looks like Windows absolute paths can start with a leading `\` https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats

Does that clarify? (from what I understand `\\` == `/` on windows?)

Reply:
Oops, I'd not noticed the backticks around the 2nd line - You only need to escape backslashes in double quotes in JSON. This PR can be closed! :) 

The leading slash makes sense to me after reading that doc - Go will translate forward slashes to Windows-specific backslashes - so should work fine!

Reply:
👍